### PR TITLE
introduce "Not now" task option and move skip to not now modal

### LINKED ISSUE #2014: move skip task option in new "Now now" modal
As mentioned in issue #1866, remove the "Skip" and "Reschedule" task options and add a new "Not Now" option. Move the "Skip" option to the "Not Now" modal, and only display it if the task is related to a budget.

@tijlleenders, I've created this separate issue to break down the skip implementation into more manageable smaller tasks.

### PR DESCRIPTION
**Pull request by vinaybadgujar102**
resolves #2014 

P.S.: The Goal/Task Action modal code needs to be refactored. The current implementation makes it difficult to maintain and apply styles. Will create a new issue for it.

### CODE REVIEW

**Review by tijlleenders — APPROVED** [review]
Looks good.

**Comment by vinaybadgujar102** [comment]
@Tushar-4781 ready for review

File: src/components/MyTimeComponents/NotNow/NotNowModal.tsx [hunk 1]
```diff
@@ -47,9 +69,15 @@ const Reschedule = () => {
...
+            <ActionDiv label="Skip" icon="Clock" />
+          </div>
+        )}
       </div>
     </ZModal>
   );
 };
 
-export default Reschedule;
+export default NowNowModal;
```
Comment (Tushar-4781) [inline src/components/MyTimeComponents/NotNow/NotNowModal.tsx:83]: what's this ?

**Review by Tushar-4781 — CHANGES_REQUESTED** [review]

**Review by Tushar-4781 — APPROVED** [review]

**Review by tijlleenders — APPROVED** [review]
I'm not sure about the wording in the 'Not now' modal. I'm inclined to use 'postpone' but if you 'postpone' for 1h it could also be scheduled even earlier in the day - as you on have indicated the current slot ia not convenient...

Anyway. This is mergeable. We'll use this for a while and see.

**Pull request merged**